editor.planx.uk/src/pages/FlowEditor/components/Settings/DesignSettings.tsx

```diff
@@ -13,7 +13,7 @@ import InputRowLabel from "ui/InputRowLabel";
 import OptionButton from "ui/OptionButton";
 import PublicFileUploadButton from "ui/PublicFileUploadButton";
 
-const TextInput: React.FC<{
+export const TextInput: React.FC<{
```

Comment:
I've added `export` here but this is genuinely unused. Ticket here to tidy up these unused `/Settings` components.